common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -853,6 +864,7 @@ object CelebornConf extends Logging {
       .doc("Push buffer queue size for a task. The maximum memory is " +
         "`celeborn.push.buffer.max.size` * `celeborn.push.queue.capacity`, " +
         "default: 64KiB * 512 = 32MiB")
+      .version("0.2.0")
```

Comment:
duplicated version assignment

Reply:
Get it, will keep the order like doc().version()

Reply:
Seems CelebornConf's methods calling order is not consistent when building ConfigEntry of various config. I will delete duplicated version in this pr, maybe we can sort out all builder's calling order in other pr.

Reply:
yea, either order is fine, as long as consistent